calculator/main.py

```diff
@@ -26,8 +33,17 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
-
+    for i in params:
+        if not isinstance(i, (int,float)):
+            raise InvalidParams("Given params are invalid.")
```

Comment:
Good! Can you guys think a more Pythonic solution? Maybe with a list comprehension?